src/device-config.ts

```diff
@@ -634,9 +634,7 @@ async function isVPNEnabled(): Promise<boolean> {
 }
 
 async function setVPNEnabled(value?: string | boolean) {
-	const v = checkTruthy(value || true);
-	const enable = v != null ? v : true;
-
+	const enable = checkTruthy(value || true);
```

Comment:
I assume this is meant to be only in the case that value is undefined? In which case it'd be better to use `??`, or even better use
```ts
async function setVPNEnabled(value?: string | boolean = true) {
```

Reply:
I like the signature change better then having this inside the function.